src/theme/MDXComponents.tsx

```diff
@@ -137,6 +139,12 @@ const StyledLink: React.FC<React.PropsWithChildren<ComponentProps<"a">>> = ({
     );
 };
 
+const Image: React.FC<React.PropsWithChildren<ComponentProps<"img">>> = ({
+  ...props
+}) => {
+  return <img {...props} className={clsx(props.className, styles.img)} />
```

Comment:
Shouldn't this be the proper Image component?

Reply:
I thought of it but it'd be better to keep it in a separate PR/issue.